ios/Exponent/Kernel/Services/EXErrorRecoveryManager.h

```diff
@@ -4,11 +4,18 @@
 //  pass error recovery info to an experience which just reloaded.
 //
 
-#import "EXErrorRecovery.h"
 #import "EXKernelService.h"
 
+// TODO: Remove once SDK34 is phased out
```

Comment:
Here we also just removed the recovery props memory management from the class — the rest is still valid and can be used. 🙂 (My suggestion is to remove `TODO:` note)